diff --git a/node_modules/cbor/lib/cbor.js b/node_modules/cbor/lib/cbor.js
index 0640b7a..fd33315 100644
--- a/node_modules/cbor/lib/cbor.js
+++ b/node_modules/cbor/lib/cbor.js
@@ -1,7 +1,7 @@
 'use strict';
 
 const Commented = require('./commented');
-const Diagnose = require('./diagnose');
+const Diagnose = { diagnose: () => {} };
 const Decoder = require('./decoder');
 const Encoder = require('./encoder');
 const Simple = require('./simple');
